components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -1329,4 +1334,32 @@ void ConversationDriver::SendFeedback(
                                   : std::nullopt,
                               std::move(on_complete));
 }
+
+void ConversationDriver::ModifyConversation(uint32_t turn_index,
+                                            const std::string& new_text) {
+  if (turn_index >= chat_history_.size()) {
+    return;
+  }
+
+  auto& turn = chat_history_.at(turn_index);
+  if (turn->character_type == CharacterType::ASSISTANT) {  // not supported yet
+    return;
+  }
+
+  std::string sanitized_input = new_text;
+  engine_->SanitizeInput(sanitized_input);
```

Comment:
Intentionally called here so we can check here, I've added a check to avoid another sanitization happening in https://github.com/brave/brave-core/commit/26769e4103cbe1d3d64a756c8fa49a214643e1ca.